testbeam_analysis/tools/plot_utils.py

```diff
@@ -632,6 +826,58 @@ def plot_events(input_tracks_file, event_range, dut, max_chi2=None, output_pdf_f
     output_pdf_file : string
         Filename of the output PDF file. If None, the filename is derived from the input file.
     '''
+
+    if gui:
+        with tb.open_file(input_tracks_file, "r") as in_file_h5:
+            fitted_tracks = False
+            try:  # data has track candidates
+                table = in_file_h5.root.TrackCandidates
+            except tb.NoSuchNodeError:  # data has fitted tracks
+                table = in_file_h5.get_node(in_file_h5.root, name='Tracks_DUT_%d' % dut)
+                fitted_tracks = True
+
+            n_duts = sum(['charge' in col for col in table.dtype.names])
+            array = table[:]
+            tracks = testbeam_analysis.tools.analysis_utils.get_data_in_event_range(array, event_range[0], event_range[-1])
+            if tracks.shape[0] == 0:
```

Comment:
The intention was to be able to plot certain event. Since it was never (?) used like that I am fine with defining `n_events`